apps/charterafrica/src/payload/blocks/Spotlight.js

```diff
@@ -0,0 +1,120 @@
+import link from "../fields/link";
+
+const Spotlight = {
+  slug: "spotlight",
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      label: {
+        en: "Title",
+        fr: "Titre",
+        pt: "Título",
+      },
+      required: true,
+      localized: true,
+    },
+    {
+      name: "items",
+      type: "array",
+      minRows: 1,
+      required: true,
+      fields: [
+        {
+          name: "category",
+          type: "text",
+          label: {
+            en: "Category",
+            fr: "Catégorie",
+            pt: "Categoria",
+          },
+          required: true,
+          localized: true,
+        },
+        {
+          name: "item",
+          type: "group",
+          label: {
+            en: "Item",
+            fr: "Objet",
+            pt: "Item",
+          },
+          required: true,
+          fields: [
+            {
+              name: "title",
+              type: "text",
+              label: {
+                en: "Title",
+                fr: "Titre",
+                pt: "Título",
+              },
+              required: true,
+              localized: true,
+            },
+            {
+              name: "image",
+              label: {
+                en: "Image",
+                fr: "Image",
+                pt: "Imagem",
+              },
+              type: "upload",
+              relationTo: "media",
+              required: true,
+              filterOptions: {
+                mimeType: { contains: "image" },
+              },
+            },
+            {
+              name: "topic",
+              type: "text",
+              label: {
+                en: "Topic",
+                fr: "Sujet",
+                pt: "Tópico",
+              },
+              required: true,
+              localized: true,
+            },
+            {
+              name: "excerpt",
+              type: "text",
+              label: {
+                en: "Excerpt",
+                fr: "Extrait",
+                pt: "Excerto",
+              },
+              required: true,
+              localized: true,
+            },
+            {
+              name: "date",
+              type: "date",
+              label: {
+                en: "Date",
+                fr: "Date",
+                pt: "Encontro",
+              },
+              required: true,
+              localized: true,
+            },
+            {
+              name: "link",
+              type: "group",
+              label: {
+                en: "Link",
+                fr: "Lien",
+                pt: "Link",
+              },
+              required: true,
+              fields: [link({})],
```

Comment:
```suggestion
              fields: [link({ disableLabel: true })],
```

Since the link is for the _card_, we don't need link label.